docs/src/13_annex.adoc

```diff
@@ -0,0 +1,80 @@
+ifndef::imagesdir[:imagesdir: ../images]
+
+[[section-annex]]
+== Annex
+=== Functional Requirements
+==== Users Sign up.
+[none or no-bullet]
+* FR-USU 1. The system will offer to an unregistered user a registration form.
+* FR-USU 2. The system will request the necessary data to register the new user.
+[none or no-bullet]
+** FR-USU 2.1. The system will request the following data to register the new user:
+[none or no-bullet]
+*** FR-USU 2.1.1. Username.
+*** FR-USU 2.1.2. Email address.
+*** FR-USU 2.1.3. Password.
+*** FR-USU 2.1.4. All data fields are mandatory.
+* FR-USU 3. When any value entered by the new user is invalid, the user will receive a message notifying the error and will not be able to register.
+* FR-USU 4. When all the values entered by the new user are valid, the system will check if the user is already registered in the persistence system.
+[none or no-bullet]
+** FR-USU 4.1. When a user with the same data is already found in the persistence system, the user will not be able to create a new account since it already exists.
+** FR-USU 4.2. When no user matching the data is found in the persistence system.
+[none or no-bullet]
+*** FR-USU 4.2.1. The new user will be registered in the system.
```

Comment:
Why the separation between 4.2 and 4.2.1? If the latter is the only consequence of the former, I don't see why they should be separated.